src/features/events/l10n/messageIds.ts

```diff
@@ -125,6 +125,12 @@ export default makeMessages('feat.events', {
     reminded: m('Reminded'),
     shifts: m('Shifts'),
     signups: m('Signups'),
+    surplusAvailable: m<{ numOthers: number }>(
+      '{numOthers, plural, =1 { + 1 more participant} other {+ # more participants}}'
+    ),
+    surplusSigned: m<{ numOthers: number }>(
+      '{numOthers, plural, =1 { + 1 more signed up} other {+ # more signed up}}'
+    ),
```

Comment:
Are these used anymore?

Reply:
No, will remove them. Name of element noted! It was called CalendarEventModal in the Figma sketch so I just went with that